src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,52 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.LocalDate;
+import duke.main.Constant;
+
+public class Event extends Task {
+    public final LocalDateTime dateTime;
```

Comment:
```suggestion
    public final LocalDateTime DATETIME;
```

Since it is a constant